back/Gemfile.lock

```diff
@@ -189,14 +189,6 @@ PATH
       rails (~> 6.1)
       verification
 
-PATH
-  remote: engines/commercial/id_gent_rrn
-  specs:
-    id_gent_rrn (0.1.0)
-      httparty (~> 0.16.2)
-      rails (~> 6.1)
-      verification
-
```

Comment:
Shouldn't be removed, not the latest ee?

Reply:
Ah yeah, that's very likely. Thanks for spotting that one.